src/cascadia/CascadiaPackage/Package-Dev.appxmanifest

```diff
@@ -26,7 +26,7 @@
   </Properties>
 
   <Dependencies>
-    <TargetDeviceFamily Name="Windows.Desktop" MinVersion="10.0.18362.0" MaxVersionTested="10.0.18362.0" />
+    <TargetDeviceFamily Name="Windows.Desktop" MinVersion="10.0.18362.0" MaxVersionTested="10.0.19041.0" />

```

Comment:
In theory, we should be using a crazy high number for this... like 12.0.0.0 or something. This is a field that the app platform uses to determine what features an app is likely to want, and it is completely undocumented as to what versions unlock what features.

Reply:
Are you sure that would be a good idea? `MaxVersionTested` is being used for switching quirks on/off and setting it to 12.0.0.0 might break our forward compatibility.

Reply:
Since we all run builds of Windows that are newer than the general public, I think I'd rather we experience leading-edge breakage so that we can go knock on doors and ask about bugs :smile:

Make sense?